learners-guild/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+echo "Updating Echo Chat Dependencies"
+
+echo "Git: Pulling repos"
+echo "Git: Pulling rocketchat-ls-sso"
+cd ../rocketchat-lg-sso
+git pull origin master
+
+echo "Git: Pulling rocketchat-ls-slash-commands"
+cd ../rocketchat-lg-slash-commands
+git pull origin master
+
+echo "Meteor: Removing packages"
+cd ../echo-chat
+echo "Meteor: Removing learnersguild:rocketchat-lg-sso"
+meteor remove learnersguild:rocketchat-lg-sso
+echo "Meteor: Removing learnersguild:rocketchat-lg-sso"
+meteor remove learnersguild:rocketchat-lg-slash-commands
+
+echo "Creating package symlinks"
+cd packages
+ln -s ../../rocketchat-lg-sso .
+ln -s ../../rocketchat-lg-slash-commands .
+
```

Comment:
Assuming that we always want things symlinked prevents us from testing installing the package from Atmosphere on our local machines. Also am absolutely against this.